### ISSUE TITLE
Windows 2013 Mouse Point

### ISSUE THREAD

**Issue by rheid** [dorianj/CoRD#45]
Sometimes if I connect to Windows 2012 Server the mouse pointer get's lost. It comes back after a lot of movement on the desktop.

**Comment by greeneg** [comment]
Agreed that the patch from JonnyBergdahl fixes this issue, with the caveat that the pointer seems to stay as the arrow and hourglass. Since the server code cannot be changed, being proprietary MS, the client MUST be modified to deal with this. Please review and merge to at least have a VISIBLE cursor in 2012 and above servers.
